scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowSCollectionFunctions.scala

```diff
@@ -75,7 +75,7 @@ private class PredictDoFn[T, V](graphBytes: DistCache[Array[Byte]],
       val outTensors = runner.run()
       try {
         import scala.collection.breakOut
-        c.output(outFn(input, (fetchOp zip outTensors.asScala)(breakOut)))
+        c.output(outFn(input, (fetchOp zip outTensors.asScala) (breakOut)))
```

Comment:
?

Reply:
@ravwojdyla do you have a scalastyle scheme for intellij? This formatting (and the docs style below) is from intellij reformat feature, using [Spotify checkstyle config](https://github.com/spotify/spotify-checkstyle-config)

Reply:
@yonromai https://github.com/spotify/scio/wiki/Style-Guide